documentation/modules/cnv-prerequisites.adoc

```diff
@@ -7,12 +7,7 @@
 = {virt} prerequisites
 
 [role="_abstract"]
-The following prerequisites apply to migrations from one {virt} cluster to another:
-
-* Both the source and destination {virt} clusters must have the same version of {project-first} installed.
-* The source cluster must use {virt} 4.16 or later.
-* Migration from a later version of {virt} to an earlier one is not supported.
-* Migration from an earlier version of {virt} to a later version is supported if both are supported by the current version of {project-short}. For example, if the current version of {virt} is 4.18, a migration from version 4.16 or 4.17 to version 4.18 is supported, but a migration from version 4.15 to any version is not.
+To migrate between {virt} clusters, verify that both clusters have matching {project-first} versions and the source uses {virt} 4.16 or later. You can migrate forward to newer {virt} versions if both are compatible with your {project-short} version.
 
```

Comment:
I think the example is helpful -- it demonstrates the actual rule, which  is that  the source must be  ( current OCP version - 2) or higher, but up to you.